MLP.py

```diff
@@ -48,19 +48,19 @@ def cv_learn(n_splits, X, y, n_hiddens, layer_size, batch_size, n_ite):
     loss_func = SquaredHingeLoss()
 
     # learn best ite
-    total_losses = {'train': np.zeros(n_ite), 'val': np.zeros(n_ite)}
-    data_splits = {'X_train': [], 'X_val': [], 'y_train': [], 'y_val': []}
+    total_losses = {'subtrain': np.zeros(n_ite), 'val': np.zeros(n_ite)}
+    data_splits = {'X_subtrain': [], 'X_val': [], 'y_subtrain': [], 'y_val': []}
```

Comment:
please remove this line, and instead initialize to empty dict then populate later with a for loop over sets

Reply:
Ok will do.